tests/cases/data/source/MongoDbTest.php

```diff
@@ -86,25 +83,99 @@ public function tearDown() {
 		MockMongoPost::reset();
 	}
 
-	public function testBadConnection() {
-		$db = new MongoDb(['host' => null, 'autoConnect' => false]);
-		$this->assertException('Could not connect to the database.', function() use ($db) {
-			$db->connect();
-		});
-		$this->assertTrue($db->disconnect());
-	}
+	public function testConnectNoHost() {
+		$config = $this->_testConfig;
+		$config['host'] = '';
+		$closure = function() use ($config) {
+			$db = new MongoDb($config);
+		};
+		$this->assertException('lithium\core\ConfigException', $closure);
```

Comment:
Style: declare the closure inline `$this->assertException('lithium\core\ConfigException', function() { ... );`